tox.ini

```diff
@@ -134,6 +134,7 @@ commands =
 isolated_build = true
 passenv =
   SSH_AUTH_SOCK
+recreate = true
```

Comment:
(educational) Note that this will guarantee an up-to-date version in the metadata at the expense of populating a fresh virtualenv on each execution.

Reply:
"Also, the tox env needs to be changed to always recreate the tox virtualenv so that locally the version is updated."

I think I misunderstood the ask here...